developer/src/kmc-analyze/src/analyzer-messages.ts

```diff
@@ -1,3 +1,6 @@
+/// <reference types="@keymanapp/common-types" />
```

Comment:
Just checking: This is still required, even with the paths removed?

Reply:
Yes, for now, see comments on subsequent lines. It may be possible to do this with an `import type` declaration but I haven't figured out what is needed yet. This works, and we are still much better off without the noise in tsconfig.json...